DCPvsOID/DCPvsOID.md

```diff
@@ -71,17 +71,18 @@ The issuance, presentation and verification of decentralized claims requires a p
 *Figure 1 Issuer-Holder-Verifier Model*
 
 ### Consent and Trust
-- __Consent:__ Organizations must approve the release of their credentials. This is managed through predefined policies.
-- __Trust Relationships:__ Trust is established through secure lists of trusted issuers and participating organizations (participants), maintained by the Dataspace Governance Authority (DSGA).
+- __Trust__: The willingness of a participant (= the trustor) to engage in a risky behavior that stems from their vulnerability to the behavior of another participant (= the trustee).
```

Comment:
Source? We should include the ISO/IEC DIS 20151 definition of Trust and trustworthiness, if required.